resqpy/olio/wellspec_keywords.py

```diff
@@ -325,12 +326,15 @@ def get_well_data(
             bytes from the beginning of the file.
         column_list (List[str]): if present, each dataframe returned contains these
             columns, in this order. If None, the resulting dictionary contains only well names as keys
-            (each mapping to None rather than a dataframe). If an empty list, each dataframe contains
+            (each mapping to None rather than a dataframe). If an empty list (default), each dataframe contains
             the columns listed in the corresponding wellspec header, in the order found in the file.
-        selecting (bool): True if the column_list contains at least one column name, False otherwise.
+        selecting (bool): True if the column_list contains at least one column name, False otherwise
+            (default).
+        keep_duplicates (bool): if True (default), duplicates in the "L" column are kept, otherwise only the
```

Comment:
The duplication needs to refer to the triplet (IW, JW, L)

A horizontal well will have constant L for a few entries but IW and/or JW changing. That should not count as a duplicate.